.betterer.results

```diff
@@ -5703,6 +5703,11 @@ exports[`better eslint`] = {
       [0, 0, 0, "Unexpected any. Specify a different type.", "1"],
       [0, 0, 0, "Unexpected any. Specify a different type.", "2"]
     ],
+    "public/app/plugins/datasource/prometheus/querycache/QueryCache.ts:5381": [
```

Comment:
I had to assert any[][] as Table, I couldn't find another way around it

Reply:
The goal of betterer is to remove the checks once we've resolved the issues, but we can't do that if we keep adding new exceptions to it.

If we are comitting to having unsafe code, you must "pay the cost" of that inline, using the `// eslint-ignore-next-line` comments, rather than adding new debt here to resolve.

Reply:
`// eslint-ignore-next-line` doesn't appear to stop the betterer results from getting added in here when I run update?

Reply:
I committed the changes after adding `// eslint-ignore-next-line` to all the type assertions and running betterer -u, still looks like it's polluting this file. Do you know what I'm doing wrong?

Reply:
Sorry, correct comment is

```ts
// eslint-disable-next-line @typescript-eslint/consistent-type-assertions
const hello = {} as FolderInfo;
```

![image](https://user-images.githubusercontent.com/46142/228556920-89a20f9f-f5ef-4eb9-92a8-0365ba544eb6.png)